app/views/users/_user.html.erb

```diff
@@ -1,8 +1,4 @@
-<div>
-    <ul>
-        <li class="users-list">
-            <%= gravatar_for(user) %>
-            <%= link_to user.name, user %>
-        </li>        
-    </ul>
-</div>
+<li class="users-list">
```

Comment:
@shokogyu It is better to add the class name `users-list` to `ul`. So that it won't be redundant. And then you can add style to the `ul.users-list` class : 
```
ul.users-list {
  list-style-type;
}
```